dGame/dComponents/CharacterComponent.h

```diff
@@ -259,6 +259,8 @@ class CharacterComponent : public Component {
 	 */
 	void UpdatePlayerStatistic(StatisticID updateID, uint64_t updateValue = 1);
 
+    uint64_t GetMaxLevel() {return m_MaxLevel;};
```

Comment:
`{ return m_MaxLevel; };`

Reply:
also changed return type to match the variable.